tests/integration_tests.rs

```diff
@@ -39,6 +41,73 @@ fn bat() -> assert_cmd::Command {
     cmd
 }
 
+/// For some tests we want mocked versions of some pagers
+/// This fn returns the absolute path to the directory with these mocked pagers
+fn get_mocked_pagers_dir() -> PathBuf {
+    let cargo_manifest_dir = env::var("CARGO_MANIFEST_DIR").expect("Missing CARGO_MANIFEST_DIR");
+    Path::new(&cargo_manifest_dir)
+        .join("tests")
+        .join("mocked-pagers")
+}
+
+/// On Unix: 'most' -> 'most'
+/// On Windows: 'most' -> 'most.bat'
+fn mocked_pager(base: &str) -> String {
+    if cfg!(windows) {
+        let mut str = String::from(base);
+        str.push_str(".bat");
+        str
```

Comment:
It's often easier to write string manipulation code like this as a `format!(…)` call. Not a huge improvement here:
```rust
format("{}.bat", base)
```

Reply:
I would actually call that a huge improvement :)
Your code is way nicer, I'll change to that